lib/src/Uncertainty/Distribution/GeneralizedPareto.cxx

```diff
@@ -368,12 +376,32 @@ Scalar GeneralizedPareto::getXi() const
   return xi_;
 }
 
+/* U accessor */
+void GeneralizedPareto::setU(const Scalar u)
+{
+  if (u != u_)
+  {
+    u_ = u;
+    isAlreadyComputedMean_ = false;
+    isAlreadyComputedCovariance_ = false;
```

Comment:
hmm covariance is invariant with translating data. Is there any reason why you set it to `false` here?

Reply:
I guess it was just a copy-paste